cmd/root.go

```diff
@@ -22,13 +22,32 @@ import (
 	"time"
 
 	"github.com/spf13/cobra"
+
+	"github.com/hacbs-contract/ec-cli/internal/utils"
 )
 
+// The default log level
+var logLevel = 1
+
 // rootCmd represents the base command when called without any subcommands
 var rootCmd = &cobra.Command{
 	Use:   "ec",
 	Short: "Tool to enforce enterprise contracts",
 	Long:  `TODO: description`,
+
+	// Prevent usage instructions showing on every error
+	SilenceUsage: true,
+
+	// Initialize logging
+	PersistentPreRun: func(cmd *cobra.Command, _ []string) {
+		utils.InitLogging(logLevel)
+	},
+}
+
+func init() {
+	rootCmd.PersistentFlags().IntVarP(&logLevel, "log-level", "l", logLevel,
```

Comment:
`logLevel` here is the default value when the flag is not provided

Reply:
Yep. I think that's a common cobra pattern.